sassc/plan.sh

```diff
@@ -0,0 +1,26 @@
+pkg_origin=core
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_name=sassc
+pkg_version=3.3.6
+pkg_source=https://github.com/sass/${pkg_name}/archive/${pkg_version}.tar.gz
+pkg_shasum=3d87edeacdd0f9a21cd0bdcf0e0e40c832b21f469af80eb5e11488c66bf53840
+pkg_license=('MIT')
+pkg_description=('libsass command line driver')
+pkg_upstream_url=https://github.com/sass/sassc
+pkg_deps=(core/glibc core/gcc)
```

Comment:
GCC should be a build dep and not a runtime dep. Does the binary need libgcc\_s or whatever that one is?  If so, you probably want gcc-libs and not gcc.